README.md

```diff
@@ -19,3 +19,16 @@ The `P256Signer` contract represents a Gnosis Safe signer for a given secp256r1
 ### P256SignerFactory.sol
 
 The `P256SignerFactory` contract is a factory for P256Signers. It allows us to have deterministic addresses for a given secp256r1 public key.
+
+
+## Launch tests
```

Comment:
Maybe create a `yarn test` command that does both.
Or create a hardhat task that does both.

Wdyt ?

Reply:
Sure, will do

Reply:
Done with [9cecfe1](https://github.com/alembic-tech/p256-signer/pull/3/commits/9cecfe184ce79a28b32cde6e9eccf41f043b0648)